common/defaults-gcp.yaml

```diff
@@ -172,50 +199,75 @@ managed_node_groups:
         effect: NO_SCHEDULE
     availability_zones: us-central1-a
 
-  # Best Effort
-  - name: be-gp-x86-az1
+  - name: co-mem-arm64-az2
+    labels:
+      demeter.run/availability-sla: consistent
+      demeter.run/compute-profile: mem-intensive
+      demeter.run/compute-arch: arm64
+      demeter.run/availability-zone: az2
+    instance_type: c4a-highmem-8
+    min_size: 0
+    max_size: 1
+    desired_capacity: 0
+    disk_size_gb: 100
+    taints:
+      - key: demeter.run/availability-sla
+        value: "consistent"
+        effect: NO_SCHEDULE
+      - key: demeter.run/compute-profile
+        value: "mem-intensive"
+        effect: NO_SCHEDULE
+      - key: demeter.run/compute-arch
+        value: "arm64"
+        effect: NO_SCHEDULE
+    availability_zones: us-central1-b
+
+  # Disk Intensive
+  - name: be-di-x86-az1
     labels:
       demeter.run/availability-sla: best-effort
       demeter.run/compute-profile: general-purpose
```

Comment:
It's best-effort and general-purpose here, but consistent and disk-intensive below.